src/containers/SearchPage/components/results/SearchContent.jsx

```diff
@@ -65,14 +74,16 @@ const SearchContent = ({ content, locale, t }) => {
       }`,
     );
   };
-
+  const langIsLocale = content.supportedLanguages.filter(l => l === locale);
   return (
     <div {...searchClasses('result')}>
       <div {...searchClasses('image')}>
         <img src={url || '/placeholder.png'} alt={alt} />
       </div>
       <div {...searchClasses('content')}>
-        <div {...searchClasses('header')}>
+        <div
+          {...searchClasses('header')}
+          style={{ display: 'flex', justifyContent: 'space-between' }}>
```

Comment:
Tviler dette er best practice når det gjelder stylingen, var mest for å få posisjonen riktig i første omgang. Fant ikke hvor jeg kunne endre 'header' classen. Den har forsåvidt ingen css uansett den klassen

Reply:
Du kan vel bruke emotion slik som du har gjort på linje 31 og legge til css klassene på den stylede div'en? 
`<StyledHeaderDiv {...searchClasses('header')}>`

Du trenger vel ikke to div'er som begge har` display:flex`?